Lakat/src/SpecialCreateBranch.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
Thanks. So what is the effect of defining a SpecialCreateBranch extension of FormSpecialPage? FormSpecialPage is I guess one of the many classes that is provided by MediaWiki to help build extensions?

Reply:
In mediawiki terms, page that is created programmatically is named [special page](https://www.mediawiki.org/wiki/Manual:Special_pages). When creating special page, the class should be inherited from SpecialPage class or one of its subclasses. I chose FormSpecialPage because we needed from and that class has some work already done. Many other special pages in mediawiki inherit FormSpecialPage, e.g. SpecialPasswordReset, I look at them as inspiration.